Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm

```diff
@@ -1004,6 +1004,19 @@ - (void)_setMediaVolumeForTesting:(float)mediaVolume
     };
 }
 
+- (void)_textFragmentRangesWithCompletionHandlerForTesting:(void(^)(NSArray<NSValue *> * fragmentRanges))completionHandler
+{
+    _page->getTextFragmentRanges([completion = makeBlockPtr(completionHandler)](const Vector<WebKit::EditingRange> editingRanges) {
+        RetainPtr<NSMutableArray<NSValue *>> resultRanges = [NSMutableArray array];
+        for (auto editingRange : editingRanges) {
+            NSRange resultRange = editingRange;
+            if (resultRange.location != NSNotFound)
+                [resultRanges addObject:[NSValue valueWithRange:resultRange]];
+        }
+        completion(WTFMove(resultRanges.get()));
```

Comment:
No `WTFMove` here, since `resultRanges` isn't being passed in as an rvalue. `editingRanges` should also be either `const Vector&` or `Vector&&`:

```suggestion
    _page->getTextFragmentRanges([completion = makeBlockPtr(completionHandler)](const Vector<WebKit::EditingRange>& editingRanges) {
        RetainPtr<NSMutableArray<NSValue *>> resultRanges = [NSMutableArray array];
        for (auto editingRange : editingRanges) {
            NSRange resultRange = editingRange;
            if (resultRange.location != NSNotFound)
                [resultRanges addObject:[NSValue valueWithRange:resultRange]];
        }
        completion(resultRanges.get());
```